apis/r/DESCRIPTION

```diff
@@ -16,6 +16,10 @@ Authors@R: c(
            email = "dirk@tiledb.com",
            role = "aut",
            comment = c(ORCID = "0000-0001-6419-907X")),
+    person(given = "Paul",
+           family = "Hoffman",
+           email = "paul.hoffman@tiledb.com",
+           role = "aut"),
```

Comment:
Can you add my ORCID?
```suggestion
    person(given = "Paul",
           family = "Hoffman",
           email = "paul.hoffman@tiledb.com",
           role = "aut",
           comment = c(ORCID = "0000-0002-7693-8957")),
```

Reply:
@mojaveazure already added you but TIL "ORCID" :)